### PR TITLE
Feature/cv Marc Sarroca

### PR DESCRIPTION
**Pull request by msarroca**

### CODE REVIEW

File: staff/marc-sarroca/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <img
+      src="https://cdn11.bigcommerce.com/s-1ah1x75lfn/products/890/images/2579/rickpolo__21335.1504202837.500.750.jpg?c=2"
+      alt="Smiley face"
+      height="150"
+      width="150"
+    />
+    <address>
+    Rick Penciner</p>
+    <b>Date of birth:</b> 30/2/1957
+    <b>City:</b> Las Vegas</address>
```
Comment (carlosclatg) [inline staff/marc-sarroca/cv/index.html]: TAB elements inside adress.

File: staff/marc-sarroca/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  <body style="text-align: center">
+    <h1>Curriculum Vitae</h1>
+    <img
+      src="https://cdn11.bigcommerce.com/s-1ah1x75lfn/products/890/images/2579/rickpolo__21335.1504202837.500.750.jpg?c=2"
+      alt="Smiley face"
+      height="150"
+      width="150"
+    />
+    <address>
+    Rick Penciner</p>
```
Comment (carlosclatg) [inline staff/marc-sarroca/cv/index.html]: Tag not opened <p>

File: staff/marc-sarroca/cv/index.html (hunk 1 above)
Comment (msarroca) [reply staff/marc-sarroca/cv/index.html]: Done

File: staff/marc-sarroca/cv/index.html (hunk 2 above)
Comment (msarroca) [reply staff/marc-sarroca/cv/index.html]: Done

File: staff/marc-sarroca/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Curriculum</title>
+  </head>
+  <body style="text-align: center">
```
Comment (Arcoders) [inline staff/marc-sarroca/cv/index.html]: without inline styles ^_^

Comment (msarroca) [reply staff/marc-sarroca/cv/index.html]: Do it

File: staff/marc-sarroca/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      src="https://cdn11.bigcommerce.com/s-1ah1x75lfn/products/890/images/2579/rickpolo__21335.1504202837.500.750.jpg?c=2"
+      alt="Smiley face"
+      height="150"
+      width="150"
+    />
+    <address>
+      <p>Rick Penciner</p>
+      <b>Date of birth:</b> 30/2/1957 <br /><b>City:</b> Las Vegas
+    </address>
+    <a><a href="mailto:rick@pawn.com">Email</a></a>
```
Comment (gcelma) [inline staff/marc-sarroca/cv/index.html]: I think you could delete the first a tag.

Comment (msarroca) [reply staff/marc-sarroca/cv/index.html]: Do it!

**Pull request merged**